<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Music_Songs_Sales</class>
 <widget class="QDialog" name="Music_Songs_Sales">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Song Sales</string>
  </property>
  <widget class="QLabel" name="AlbumSales_NumberText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Song ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesFeb16">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales Feb '16</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SongSales_SalesBoxMay16">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>350</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SongSales_PrintButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_TitleText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Song Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesNov15">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales Nov '15</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SongSales_SalesBoxJan16">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SongSales_SaveButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Changes</string>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumTitleBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesDec15">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales Dec '15</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SongSales_SalesBoxDec15">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesOct15">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales Oct '15</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SongSales_SalesBoxAug15">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="SongSales_TitleSearch">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesJan16">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales Jan '16</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SongSales_SalesBoxJul15">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesJun15">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales June '15</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SongSales_SalesBoxSep15">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="SongSales_SalesBoxApr16">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SongSales_BackButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SongSales_SalesBoxJun15">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="SongSales_SalesBoxOct15">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="SongSales_SalesBoxFeb16">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesAug15">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales Aug '15</string>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesSep15">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales Sept '15</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SongSales_SearchButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesJul15">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales July '15</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SongSales_NumberSearch">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumNumberBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="SongSales_SalesBoxMar16">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesMar16">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales Mar '16</string>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesApr16">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales Apr '16</string>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesMay16">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales May '16</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SongSales_SalesBoxNov15">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_MainText">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Song Sales</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SongSales_BackButton</sender>
   <signal>clicked()</signal>
   <receiver>Music_Songs_Sales</receiver>
   <slot>BackButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>21</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>10</x>
     <y>731</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SongSales_SearchButton</sender>
   <signal>clicked()</signal>
   <receiver>Music_Songs_Sales</receiver>
   <slot>ViewSales()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>483</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SongSales_PrintButton</sender>
   <signal>clicked()</signal>
   <receiver>Music_Songs_Sales</receiver>
   <slot>Print()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SongSales_SaveButton</sender>
   <signal>clicked()</signal>
   <receiver>Music_Songs_Sales</receiver>
   <slot>SaveChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>565</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>BackButton()</slot>
  <slot>ViewSales()</slot>
  <slot>Print()</slot>
  <slot>SaveChanges()</slot>
 </slots>
</ui>
